warehouse/query-core/src/main/java/datawave/query/util/FuzzyAttributeComparator.java

```diff
@@ -0,0 +1,91 @@
+package datawave.query.util;
+
+import com.google.common.collect.Sets;
+import datawave.query.attributes.Attribute;
+import datawave.query.attributes.Attributes;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class FuzzyAttributeComparator {
+    
+    // this is a utility and should never be instantiated
+    private FuzzyAttributeComparator() {
+        throw new UnsupportedOperationException();
+    }
+    
+    public static boolean singleToSingle(final Attribute existingAttribute, final Attribute newAttribute) {
```

Comment:
Let's standardize the argument naming convention here. The method names are keeping to `single` and `multiple` but the args are `existing`, `new`, `single`, and `multiple`. You could make this a simple `left`/`right` or `leftSingle` and `rightSingle`. Or perhaps `attr` for single Attribute and `attrs` for an instance of Attributes. Just make the args consistent among themselves.

Reply:
Done and done.